json2graphite.rb

```diff
@@ -153,8 +153,13 @@ def metrics(data, timestamp, parent_key = nil)
 OptionParser.new do |opt|
   opt.on('--pattern PATTERN') { |o| $options[:pattern] = o }
   opt.on('--netcat HOST') { |o| $options[:host] = o }
+  opt.on('--port PORT') { |p| $options[:port] = p }
 end.parse!
 
+if $options[:port].nil?
```

Comment:
This doesn't actually seem to work.  I only tested passing in a new port previously.  Now I tested trying to run view-in-graphana.sh and it doesn't work.